utils/formats/formatFontFace.js

```diff
@@ -0,0 +1,47 @@
+/**
+ *
+ * @param allTokens
+ * @param options
+ * @returns {*}
+ *
+ * @see @see {@link https://github.com/amzn/style-dictionary/blob/main/examples/advanced/font-face-rules/sd.config.js#L18}
+ */
+export const formatFontFace = (allTokens, options) => {
+    const fontPathPrefix = options.fontPathPrefix || '../';
+
+    // https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/src
+    const formatsMap = {
+        'woff2': 'woff2',
+        'woff': 'woff',
+        'ttf': 'truetype',
+        'otf': 'opentype',
+        'svg': 'svg',
+        'eot': 'embedded-opentype'
```

Comment:
do we actually need these formats? from my (limited!) reading, I don't think these are recommended/required any more...

https://stackoverflow.com/a/37091681